src/blackcore/fsd/fsdclient.cpp

```diff
@@ -1282,36 +1282,36 @@ namespace BlackCore::Fsd
-        switch (messageType)
-        {
-        case MessageType::VisualPilotDataUpdate: dataUpdate = VisualPilotDataUpdate::fromTokens(tokens); break;
-        case MessageType::VisualPilotDataPeriodic: dataUpdate = VisualPilotDataPeriodic::fromTokens(tokens).toUpdate(); break;
-        case MessageType::VisualPilotDataStopped: dataUpdate = VisualPilotDataStopped::fromTokens(tokens).toUpdate(); break;
-        default: qFatal("Precondition violated"); break;
-        }
-        const CCallsign callsign(dataUpdate.sender(), CCallsign::Aircraft);
-
-        CAircraftSituation situation(
-            callsign,
-            CCoordinateGeodetic(dataUpdate.m_latitude, dataUpdate.m_longitude, dataUpdate.m_altitudeTrue),
-            CHeading(dataUpdate.m_heading, CHeading::True, CAngleUnit::deg()),
-            CAngle(dataUpdate.m_pitch, CAngleUnit::deg()),
-            CAngle(dataUpdate.m_bank, CAngleUnit::deg()));
+    void CFSDClient::handleVisualPilotDataUpdate(const QStringList & /*tokens*/, MessageType /*messageType*/)
+    {
+        // VisualPilotDataUpdate dataUpdate;
+        // switch (messageType)
+        // {
+        // case MessageType::VisualPilotDataUpdate: dataUpdate = VisualPilotDataUpdate::fromTokens(tokens); break;
+        // case MessageType::VisualPilotDataPeriodic: dataUpdate = VisualPilotDataPeriodic::fromTokens(tokens).toUpdate(); break;
+        // case MessageType::VisualPilotDataStopped: dataUpdate = VisualPilotDataStopped::fromTokens(tokens).toUpdate(); break;
+        // default: qFatal("Precondition violated"); break;
+        // }
+        // const CCallsign callsign(dataUpdate.sender(), CCallsign::Aircraft);
+
+        // CAircraftSituation situation(
+        //     callsign,
+        //     CCoordinateGeodetic(dataUpdate.m_latitude, dataUpdate.m_longitude, dataUpdate.m_altitudeTrue),
+        //     CHeading(dataUpdate.m_heading, CHeading::True, CAngleUnit::deg()),
+        //     CAngle(dataUpdate.m_pitch, CAngleUnit::deg()),
+        //     CAngle(dataUpdate.m_bank, CAngleUnit::deg()));
 
         // not used
         // situation.setVelocity(CAircraftVelocity(
         //    dataUpdate.m_xVelocity, dataUpdate.m_yVelocity, dataUpdate.m_zVelocity, CSpeedUnit::m_s(),
         //    dataUpdate.m_pitchRadPerSec, dataUpdate.m_bankRadPerSec, dataUpdate.m_headingRadPerSec, CAngleUnit::rad(), CTimeUnit::s()));
 
         // Ref T297, default offset time
-        situation.setCurrentUtcTime();
-        const qint64 offsetTimeMs = receivedPositionFixTsAndGetOffsetTime(situation.getCallsign(), situation.getMSecsSinceEpoch());
-        situation.setTimeOffsetMs(offsetTimeMs);
+        // situation.setCurrentUtcTime();
+        // const qint64 offsetTimeMs = receivedPositionFixTsAndGetOffsetTime(situation.getCallsign(), situation.getMSecsSinceEpoch());
+        // situation.setTimeOffsetMs(offsetTimeMs);
 
-        emit visualPilotDataUpdateReceived(situation);
+        // emit visualPilotDataUpdateReceived(situation);
```

Comment:
I generally prefer using `#if 0` to comment out large blocks of code. In this case in particular, it would help avoid the ambiguity due to lines 1304-1309 already being commented out and it not being clear that that portion was already commented out prior to this commenting out.

Reply:
Changed